packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -260,18 +260,16 @@ const MetadataLabeling = (props: Props) => {
     const onSubmit = async (value: string | undefined | null) => {
         isSubscribedToSubmitResult.current = props.payload.defaultValue;
         setPending(true);
-        const result = await addMetadata({
+        await addMetadata({
```

Comment:
suite code should not be affected

Reply:
changes was made due to unnecessary packages, reverted in 68c7a59